### ISSUE TITLE
Dyadic notation

### ISSUE THREAD

**Issue by moorepants** [moorepants/learn-multibody-dynamics#181]
I read a paper that used $\underline{I}$ for a dyadic. If I stick with $\bar{v}$ for vectors, then the underline makes for a better fit. The unit dyadic could still use a special symbol (analogous to $\hat{u}$).

I could also define $\hat{n}_{xx} = \hat{n}_x \otimes \hat{n}_x$ for shorter notation on unit dyadics, but maybe that is confusing because you would think it is a unit vector with a special subscript.

**Comment by Peter230655** [comment]
The book Dynamics... by Roithmayr and Hodges seems to use the 'underline' notation, with capital letters, see p76 ff of this book

**Comment by moorepants** [comment]
Do they use anything for a unit dyadic?

**Comment by Peter230655** [comment]
Yes, they call it $\underline{U}$

**Comment by moorepants** [comment]
Ok, I was wondering if they had something different. I use hat for unit vectors, a different symbol than the bar for my vectors.

**Comment by moorepants** [comment]
I guess I could do $\hat{\underline{u}}$.

**Comment by Peter230655** [comment]
For vectors, they use boldface only, like $bm{v}-r$ for the r-th partial velocity.

**Comment by moorepants** [comment]
Yes, I'm aware of that. I can't write bold letters on the chalkboard, so I prefer to avoid it.
